### ISSUE TITLE
include_status being sent when making a FetchInstitutionsAsync request

### ISSUE THREAD

**Issue by stijwilliams** [viceroypenguin/Going.Plaid#42]
It appears that the institutions options are unique depending on the endpoint you are calling.  Example, I'm not able to call FetchInstitutionsAsync while providing the `IncludeOptionalMetaData` option because the `options.include_status` is also being sent.

Example code snippit of what I'm trying to do.

```c#
reqSearch.Query = keyword.ToUpper();
reqSearch.Options = new Going.Plaid.Options.InstitutionSearchOptions {
    IncludeOptionalMetadata = true
};
var resultsSearch = await this._client.FetchInstitutionsAsync(reqSearch);
```

But, I receive the following for my efforts.

```
    "StatusCode": "BadRequest",
    "RequestId": null,
    "DocumentationUrl": "https://plaid.com/docs/?ref=error#invalid-request-errors",
    "ErrorCode": "UnknownFields",
    "ErrorType": "InvalidRequest",
    "Exception_RequestId": "MJDHcpFPpzbqAd9",
    "SuggestedAction": null,
    "ErrorMessage": "the following fields are not recognized by this endpoint: options.include_status"
```

**Comment by stijwilliams** [comment]
Quick link to the API documentation reference for available options when doing an institution search.
https://plaid.com/docs/api/institutions/#institutionssearch

**Comment by viceroypenguin** [comment]
Thanks for pointing that out. I'll get an update out shortly, hopefully tomorrow or Tuesday!

**Comment by viceroypenguin** [comment]
Tentatively released under v3.0.0-preview1. I have other breaking changes to include in v3.0, so not a full production release yet. Hope this helps! Let me know if you need anything else.
